multidiff/command_line_interface.py

```diff
@@ -75,6 +89,11 @@ def make_parser():
 		const='html',
 		default='ansi',
 		help='use html for colors instead of ansi codes')
+
+	parser.add_argument('-w', '--width',
+		dest='width',
+		default='82',
```

Comment:
I think there's a misunderstanding of what feature is wanted, I've added a comment to the issue, where we can discuss this https://github.com/juhakivekas/multidiff/issues/3#issuecomment-500329052